bin/pytorch_inference/Main.cc

```diff
@@ -290,14 +293,14 @@ int main(int argc, char** argv) {
         return EXIT_FAILURE;
     }
 
-    if (numLibTorchThreads != -1) {
-        at::set_num_threads(numLibTorchThreads);
-    }
-    if (numLibTorchInterOpThreads != -1) {
-        at::set_num_interop_threads(numLibTorchInterOpThreads);
-    }
+    at::set_num_threads(inferenceThreads);
```

Comment:
This can currently be set to 0 or a huge number.  Probably we should constrain it into the range [1, `std::thread::hardware_concurrency`].

Reply:
Ah yes, capping. Will add this.